.env.dev

```diff
@@ -23,6 +23,7 @@ HACK_LOCAL_ROOT_PARENT=/tmp
 WEBAPP_URL=http://localhost:8000/
 API_URL=/api/v1/
 INTERNAL_API_HOST=airbyte-server:8001
+#CONNECTOR_BUILDER_API_HOST=airbyte-connector-builder:8003 #FIXME: Uncomment this when enabling the connector-builder
```

Comment:
this is commented out for now because this is stubbed?

Reply:
yes. I doubt there would be any harm to exposing the env variable, but we can just turn this on at the same time as we enable the server